compose/ccd-next-hearing-date-updater.yml

```diff
@@ -0,0 +1,32 @@
+---
+version: '2.4'
+
+services:
+  ccd-next-hearing-date-updater:
+    image: "${CCD_NEXT_HEARING_UPDATER_LOCAL-hmctspublic.azurecr.io/}ccd/next-hearing-date-updater:${CCD_NEXT_HEARING_UPDATER_TAG:-latest}"
+    environment:
+      IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      CCD_NEXT_HEARING_DATE_UPDATER_SERVICE_IDAM_CLIENT_SECRET: AAAAAAAAAAAAAAAA
+      IDAM_NEXT_HEARING_DATE_SYSTEM_USER: "${CCD_NEXT_HEARING_DATE_USERNAME:-next.hearing.day.idam.system.user@gmail.com}"
+      IDAM_NEXT_HEARING_DATE_SYSTEM_PASSWORD: "${CCD_NEXT_HEARING_DATE_PASSWORD}"
```

Comment:
IDAM_NEXT_HEARING_DATE_UPDATER_SYSTEM_PASSWORD ??

Reply:
I could change it but this would lead to change the original HMAN-320/326. All names are exactly the same from this pr. I think that first we need all agreed in that refactor on HMAN-320/326.

Reply:
Closing as this will work as is now that the username has been updated.